api/src/main/java/com/epam/pipeline/dao/metadata/CategoricalAttributeDao.java

```diff
@@ -72,29 +86,60 @@ public boolean updateCategoricalAttributes(final List<CategoricalAttribute> dict
     @Transactional(propagation = Propagation.MANDATORY)
     public boolean insertAttributesValues(final List<CategoricalAttribute> dict) {
         final MapSqlParameterSource[] values = dict.stream()
-            .flatMap(entry -> entry.getValues()
```

Comment:
Could you please remove `this` on lines 82 & 83 (method `updateCategoricalAttributes `)